ci/Jenkinsfiles/release.groovy

```diff
@@ -271,6 +282,14 @@ pipeline {
       script {
         if (env.DRY_RUN != "true") {
           currentBuild.description = "Release ${RELEASE_VERSION} from build ${params.BUILD_VERSION}"
+          slackSend(channel: "${SLACK_CHANNEL}", color: "good", message: "Successfully released ${RELEASE_VERSION} from build ${params.BUILD_VERSION}")
+        }
+      }
+    }
+    unsuccessful {
+      script {
+        if (env.DRY_RUN != "true") {
+          slackSend(channel: "${SLACK_CHANNEL}", color: "danger", message: "Failed to promote ${RELEASE_VERSION} from build ${params.BUILD_VERSION}: ${BUILD_URL}")
```

Comment:
Stick with "release" instead of "promote"?

Reply:
ah sorry 🤦